module2/src/com/epam/onedimensionalarrays/CalculateSumOfArray.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.onedimensionalarrays;
+
+public class CalculateSumOfArray {
```

Comment:
Rename this class according to name conventions - https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Class%20names%20should%20be%20nouns,such%20as%20URL%20or%20HTML).
```suggestion
public class SumCalculatorOfNumbersDivisibleByK {
```